src/libs/OptionsListUtils.js

```diff
@@ -466,6 +466,8 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
     if (searchValue
         && recentReportOptions.length === 0
         && personalDetailsOptions.length === 0
+        // eslint-disable-next-line no-use-before-define
+        && !isCurrentUser({login: searchValue})
```

Comment:
As far as I can see, the lint suppression is required because you created an Object to pass into `isCurrentUser`? If not please ignore me 🙃 

But if so, I think a better solution would be to modify the `isCurrentUser` function. There seems no point in taking the user object as a param, when instead we could just take the `userLogin` param. It should be a simple change that clears up the usage in `OptionsListUtils`. [Here](https://github.com/Expensify/App/blob/8b10c1c1a0267eb66ef8fdc6e0049f9b101836a2/src/libs/OptionsListUtils.js#L731-L753) is the function.

Reply:
Oh, it's referring to the function. Ignore the above then.

Reply:
So I still think the lint suppression is unnecessary though. Could you move the function up within the OptionsListUtils file so that it is above `getOptions`. That should prevent the lint warning.